### PR TITLE
Implemented Get Release Build Info Call

### PR DESCRIPTION
**Pull request by k127**
_Release Build Info_ contains the list of order numbers which are “Ready for Next Release” in the project.
https://www.getdonedone.com/api/
(Changes have not been tested)

### CODE REVIEW

File: src/Project.php [hunk 1]
```diff
@@ -133,6 +133,16 @@ public function releaseBuilds()
     }
 
     /**
+     * Get release builds of this project
+     *
+     * @return ReleaseBuildInfo
```
Comment (manavo) [inline src/Project.php]: Not sure this is quite right? It looks like it is returning the raw data from the client, which probably won't be an instance of ReleaseBuildInfo?

**Comment by manavo** [comment]
Thanks for the PR @k127! I'm a bit confused about the `src/ReleaseBuildInfo.php` class, can't see it actually used anywhere? And there's no constructor for it to be initialised/populated?

**Comment by k127** [comment]
Hi @manavo, I now added the object population – hope it's the right way. Sorry for the rather poor attempt, I'm just learning. What's still missing is the method test in `tests/ProjectTest.php`. I didn't know how to do that.

**Comment by manavo** [comment]
Sorry for the delayed merge @k127!

**Pull request merged**